content/en/language_clients/javascript/overview.md

```diff
@@ -0,0 +1,73 @@
+---
+type: docs
+category: JavaScript
+title: JavaScript Client Overview
+weight: 5
+---
+
+[sigstore-js](https://github.com/sigstore/sigstore-js) is a collection of [javascript libraries](#additional-packages) for interacting with Sigstore.
+
+The main package,[`sigstore`](https://www.npmjs.com/package/sigstore), is a JavaScript library for generating and verifying Sigstore signatures. One of the intended uses is to sign and verify npm packages but it can be used to sign and verify any file.
+
+Full project documentation can be found in the [sigstore-js](https://github.com/sigstore/sigstore-js#sigstore-js---) project README and in each [package README](https://github.com/sigstore/sigstore-js/tree/main/packages).
+
+## Features
+
+- Support for signing using an OpenID Connect identity
+- Support for publishing signatures to a [Rekor](../../logging/overview/) instance
+- Support for verifying Sigstore bundles
+
+## Installation
+
+[`sigstore`](https://www.npmjs.com/package/sigstore) requires Node.js version >= 18.17.0.
+
+To install `sigstore` run the following command:
+
+```console
+npm install sigstore
+```
+
+## Example
+
+ To use `sigstore`, import the following into your project using the following:
+
+ ```console
+ const { sign, verify } = require('sigstore');
+ ```
+
+ ```console
+ import { sign, verify } from 'sigstore';
+ ```
+
+### Sign
+
+ The following function will sign the file `foo.txt`:
+
+ ```console
+ sign('foo.txt');
```

Comment:
I want particular review for the signing and verifying examples. Node/JS is not my preferred language and I was not able to try the library out. As far as I can tell from [this](https://github.com/sigstore/sigstore-js/tree/main/packages/client#usage) I think I got at least some things right, but it is a guess to some degree.

Reply:
The argument to `sign` is a buffer (byte array) containing the message to be signed, so it may be better to show something like:

```
const bundle = await sign(Buffer.from('hello world'));
```